dynamics/src/include/VPCGDynamicsKernel.hpp

```diff
@@ -124,7 +127,9 @@ template <int DGadvection> class VPCGDynamicsKernel : public CGDynamicsKernel<DG
                     + cgA(i)
                         * (FAtm * absatm * uAtmos(i) + // atm forcing
                             FOcean * absocn * SC * uOcean(i)) // ocean forcing
-                    + params.rhoIce * cgH(i) * params.fc * vOcnRel // cor + surface
+                    - params.rhoIce * cgH(i) * params.fc * u(i) // Coriolis
```

Comment:
Is `u(i)` correct here? I would expect the Coriolis term to take the other component of the 2D velocity.

Reply:
Nice catch!

Reply:
This is fixed now. Can you re-review @timspainNERSC ?